godel_surface_detection/src/services/surface_blending_service.cpp

```diff
@@ -1003,6 +991,8 @@ void SurfaceBlendingService::visualizePaths()
   }
 
   tool_path_markers_pub_.publish(path_visualization);
+  process_planning_result_.succeeded = true;
+  process_planning_server_.setSucceeded(process_planning_result_);
```

Comment:
I was looking at what the last step in the request was "generate_motion_plan_and_PREVIEW", but I agree, it's more visible and makes more sense in the handler callback.